### PR TITLE
ZFIN-8217: RenoRepository Refactor Hibernate Queries

### PR DESCRIPTION
**Pull request by rtaylorzfin**
One note:

I left some of the hairier methods alone.  For example: getSortedRunCandidates.  Do we want to refactor those?

### CODE REVIEW

File: source/org/zfin/sequence/reno/repository/HibernateRenoRepository.java [hunk 1]
```diff
@@ -263,10 +306,7 @@ public List<RunCandidate> getSortedNonZFRunCandidates(Run run, String comparator
 
 
     public RunCandidate getRunCandidateByID(String runCandidateID) {
-        Session session = HibernateUtil.currentSession();
-        Criteria criteria = session.createCriteria(RunCandidate.class);
-        criteria.add(Restrictions.eq("zdbID", runCandidateID));
```
Comment (cmpich) [inline source/org/zfin/sequence/reno/repository/HibernateRenoRepository.java:268]: Is this method not used?

Comment (rtaylorzfin) [reply source/org/zfin/sequence/reno/repository/HibernateRenoRepository.java:268]: This method is used in a few places.  Why do you ask?

Comment (cmpich) [reply source/org/zfin/sequence/reno/repository/HibernateRenoRepository.java:268]: never mind.

**Review by cmpich — COMMENTED** [review]
Looks good.

**Review by cmpich — APPROVED** [review]

**Pull request merged**